keyboards/annepro2/keymaps/default-layer-indicators/keymap.c

```diff
@@ -0,0 +1,151 @@
+    KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PGUP, KC_PGDN, KC_TRNS,
+    KC_TRNS, KC_VOLU, KC_VOLD, KC_MUTE, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_INS, KC_DEL, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, MO(_FN2_LAYER), KC_TRNS
+),
+  /*
+  * Layer _FN2_LAYER
+  * ,-----------------------------------------------------------------------------------------.
+  * |  ~  | BT1 | BT2 | BT3 | BT4 |  F5 |  F6 |  F7 |LEDOF|LEDON| F10 | F11 | F12 |    Bksp   |
+  * |-----------------------------------------------------------------------------------------+
+  * | Tab    |  q  | UP  |  e  |  r  |  t  |  y  |  u  |  i  |  o  | PS | HOME | END |   \    |
+  * |-----------------------------------------------------------------------------------------+
+  * | Esc     |LEFT |DOWN |RIGHT|  f  |  g  |  h  |  j  |  k  |  l  | PGUP|PGDN |    Enter    |
+  * |-----------------------------------------------------------------------------------------+
+  * | Shift      |  z  |  x  |  c  |  v  |  b  |  n  |  m  |  ,  |INSRT| DEL |    Shift       |
+  * |-----------------------------------------------------------------------------------------+
+  * | Ctrl  |  L1   |  Alt  |               space             |  Alt  |  FN1  |  FN2  | Ctrl  |
+  * \-----------------------------------------------------------------------------------------/
+  *
+  */
+ [_FN2_LAYER] = KEYMAP( /* Base */
+    KC_TRNS, KC_AP2_BT1, KC_AP2_BT2, KC_AP2_BT3, KC_AP2_BT4, KC_TRNS, KC_TRNS, KC_TRNS, KC_AP_LED_OFF, KC_AP_LED_ON, KC_AP_LED_NEXT_INTENSITY, KC_AP_LED_SPEED, KC_TRNS, KC_TRNS,
+    MO(_FN2_LAYER), KC_TRNS, KC_UP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_HOME, KC_END, KC_TRNS,
+    KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PGUP, KC_PGDN, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_INS, KC_DEL, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, MO(_FN1_LAYER), MO(_FN2_LAYER), KC_TRNS
+ ),
+};
+
+// clang-format on
+const uint16_t keymaps_size = sizeof(keymaps);
+
+void matrix_init_user(void) {}
+
+void matrix_scan_user(void) {}
+
+// Code to run after initializing the keyboard
+void keyboard_post_init_user(void) {
+    // Here are two common functions that you can use. For more LED functions, refer to the file "qmk_ap2_led.h"
+
+    // annepro2-shine disables LEDs by default. Uncomment this function to enable them at startup.
+    // annepro2LedEnable();
+
+    // Additionally, it also chooses the first LED profile by default. Refer to the "profiles" array in main.c in
+    // annepro2-shine to see the order. Replace "i" with the index of your preferred profile. (i.e the RED profile is index 0)
+    // annepro2LedSetProfile(i);
+}
+
+layer_state_t layer_state_set_user(layer_state_t layer) {
+    switch (get_highest_layer(layer)) {
```

Comment:
While the name doesn't technically matter to the compiler, this is a bitmask of the layer state, so `state` is a bit more accurate and clear. 
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
    switch (get_highest_layer(state)) {
```